Do not try to generate the plugin cache at build time. We will handle this in
pkg_postinst

Index: vlc-1.1/modules/Makefile.am
===================================================================
--- vlc-1.1.orig/modules/Makefile.am
+++ vlc-1.1/modules/Makefile.am
@@ -62,16 +62,4 @@ MOSTLYCLEANFILES = stamp-cache plugins-*
 .PHONY: stamp-cache
 
 stamp-cache:
-	$(AM_V_GEN)if test "$(build)" = "$(host)"; then \
-		../bin/vlc-cache-gen$(EXEEXT) . ; \
-	else \
-		echo "Cross-compilation: cache generation skipped!" ; \
-	fi
 	$(AM_V_at)touch stamp-cache
-
-install-exec-hook:
-	if test -z "$(DESTDIR)"; then \
-		../bin/vlc-cache-gen$(EXEEXT) "$(vlclibdir)/plugins" ; \
-	else \
-		echo "Staged installation: cache generation skipped!" ; \
-	fi
